docs/standards/universal-profile/introduction.md

```diff
@@ -3,28 +3,35 @@ sidebar_label: 'Introduction'
 
-:::caution This section is a work in progress.
 :::
+## Introduction
+
+Building an identity on top of keys is almost impossible because at some point they are very likely to being leaked or lost forever. Holding assets and building reputation on these keys makes it worse. A better representation of identity would be with blockchain based accounts.
 
-> To see Universal Profiles (🆙 ) live on the [L14 testnet](../../networks/l14-testnet.md), check out [universalprofile.cloud](https://universalprofile.cloud/). It lets you easily browse the deployed profiles.
+Blockchain based accounts can change the way of interacting on-chain, they allow the usage of multiple component together. They could be used with a key Manager where you could allow any entity to execute or set some data on your profile. A social recovery contract could be used in case of keys-loss and many more component that enhance the blockchain experience.
 
-Universal Profiles are smart contract accounts, that should be used instead of simple EOAs (simple generated keys), as they have the following advantages:
+## Universal Profiles
 
-- Security is upgradeable, controlling keys or smart contracts can be changed over time, and even the whole key manager logic
-- Accounts can attach information though [ERC725Y](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-725.md#erc725y), meaning things from [profile information](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-3-UniversalProfile-Metadata.md) to protocols and other data, can be attached to your account.
-- Accounts can react to incoming assets, due to the an upgradeable [Universal Receiver](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-1-UniversalReceiver.md).
 
-A smart contract account can consist of 1 to 3 pieces:
+- **[LSP0 - ERC725Account](./01-lsp0-erc725account.md)**: This standard represents the core account contract, it consists of [ERC725X](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-725.md#erc725x) that allows you to interact with any address or smart contract on the blockchain, and [ERC725Y](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-725.md#erc725y) that allows you to store unlimited key value pairs at your account. It also contains [ERC1271](https://eips.ethereum.org/EIPS/eip-1271) to verify of messages where signed by the owner of the ERC725Account, if the owner is a KeyManager supporting ERC1271 it will delegate the call to the KeyManager. And lastly it contains the [LSP1-UniversalReceiver](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-1-UniversalReceiver.md), which allows this contract to be notified of any incoming asset. Using the UniversalReceiver Delegate logic, you can delegate the universal receiver call that an asset will do to an external contract, customizing the behaviour you want towards the asset. More on that below.     
```

Comment:
```suggestion
- **[LSP0 - ERC725Account](./01-lsp0-erc725account.md)**: This standard represents the core account contract. It consists of [ERC725X](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-725.md#erc725x) that allows you to interact with any address or smart contract on the blockchain, and [ERC725Y](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-725.md#erc725y) that allows you to store unlimited key value pairs at your account. It also contains [ERC1271](https://eips.ethereum.org/EIPS/eip-1271) to verify if messages where signed by the owner of the ERC725Account, if the owner is a KeyManager supporting ERC1271 it will delegate the call to the KeyManager. Finally, it contains the [LSP1-UniversalReceiver](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-1-UniversalReceiver.md), which allows this contract to be notified of any incoming asset. Using the UniversalReceiver Delegate logic, you can delegate the universal receiver call that an asset will do to an external contract, customizing the behaviour you want towards the asset. More on that below.     
```